src/Symfony/Bundle/WebProfilerBundle/Resources/views/Collector/workflow.html.twig

```diff
@@ -1,5 +1,127 @@
 {% extends '@WebProfiler/Profiler/layout.html.twig' %}
 
+{% block stylesheets %}
+    {{ parent() }}
+    <style>
+        :root {
+            --bg: rgb(26, 27, 30);
+            --color: #ffffff;
+            --button-bg: rgb(50, 52, 61);
+            --button-color: #ffffff;
+            --warn-bg: rgb(198, 48, 48);
+            --modal-color: rgb(155, 155, 160);
+        }
+
+        button {
```

Comment:
applying this on all buttons on the page is a no-go. It can break other parts of the design.

Reply:
if you cannot add a class on the button itself due to being in mermaid, use `.sf-mermaid button`